pkg/devfile/generator/utils.go

```diff
@@ -61,12 +62,38 @@ func convertPorts(endpoints []v1.Endpoint) []corev1.ContainerPort {
 func getResourceReqs(comp v1.Component) corev1.ResourceRequirements {
 	reqs := corev1.ResourceRequirements{}
 	limits := make(corev1.ResourceList)
-	if comp.Container != nil && comp.Container.MemoryLimit != "" {
-		memoryLimit, err := resource.ParseQuantity(comp.Container.MemoryLimit)
-		if err == nil {
-			limits[corev1.ResourceMemory] = memoryLimit
+	requests := make(corev1.ResourceList)
+	if comp.Container != nil {
+		if comp.Container.MemoryLimit != "" {
+			memoryLimit, err := resource.ParseQuantity(comp.Container.MemoryLimit)
+			if err == nil {
```

Comment:
Should we issue an error message if ParseQuantity fails?  The field will not be set in that case

Reply:
good point. updated